client/package.json

```diff
@@ -3,17 +3,25 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.4",
```

Comment:
Te biblioteki emotion gdzies uzywane sa wgl? Jak juz uzywamy scss do stylowania, to sie wszedzie tego scss trzymajmy